docs/source/getting-started/security-basics.rst

```diff
@@ -239,7 +239,7 @@ jupyterhub-session-id
 This is a random string, meaningless in itself, and the only cookie
 shared by the Hub and single-user servers.
 
-Its sole purpose is to to coordinate logout of the multiple OAuth cookies.
+Its sole purpose is to coordinate logout of the multiple OAuth cookies.
```

Comment:
And mea-culpa, because I believe I was the one to add this line in a previous PR :sweat_smile: sorry!